sdk/src/client/api/block_builder/input_selection/requirement/context_inputs.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use super::{Error, InputSelection};
+use crate::{
+    client::secret::types::InputSigningData,
+    types::block::{
+        context_input::{BlockIssuanceCreditContextInput, CommitmentContextInput, RewardContextInput},
+        output::{AccountId, DelegationOutputBuilder, Output},
+    },
+};
+
+impl InputSelection {
+    pub(crate) fn fulfill_context_inputs_requirement(&mut self) -> Result<Vec<InputSigningData>, Error> {
```

Comment:
This is at the very end of the process, right? This can't be called more than once?

Reply:
shouldn't

Reply:
now it is